src/update_agent/actor.rs

```diff
@@ -138,17 +138,49 @@ impl UpdateAgent {
         std::time::Duration::from_secs(secs.saturating_add(jitter))
     }
 
+    /// Log at INFO level how many and which deployments will be excluded from being
+    /// future update targets.
+    fn log_excluded_depls(depls: &BTreeSet<Release>, actor: &UpdateAgent) {
+        let excluded_depls_count = depls.len().saturating_sub(1); // Minus current booted deployment.
```

Comment:
It may be cleaner to just build a local vector excluding the booted deployment, and then use that for the rest of this function.